docs/api/en/loaders/ImageBitmapLoader.html

```diff
@@ -16,6 +16,13 @@ <h1>[name]</h1>
 			Unlike [page:FileLoader], [name] does not avoid multiple concurrent requests to the same URL.
 		</p>
 
+		<p>
+			Note that [page:Texture.flipY .flipY] and [page:Texture.premultiplyAlpha .premultiplyAlpha] properties of [page:Texture]
```

Comment:
I would shortly note why both properties are ignored (because unlike normal images they are needed when the bitmap is created; not at texture upload time).

Reply:
So, how about adding

... are ignored because corresponding `UNPACK_FLIP_Y_WEBGL` and `UNPACK_PREMULTIPLY_ALPHA_WEBGL` parameters for `gl.pixelStorei()` are ignored if image source is `ImageBitmap`.

Reply:
It seems more important to me to differentiate between creation and upload time of the texture. In context of `ImageBitmap`, you need these configuration parameters when creating the bitmap. In all other cases, you need them when the texture is uploaded to the GPU. That's the important bit users should aware of and the reason why setting `Texture.flipY` and `Texture.premultiplyAlpha` after the creation process are ignored.

Reply:
How about this

Note that `Texture.flipY` and `Texture.premultiplyAlpha` with `ImageBitmap` are ignored. `ImageBitmap` needs these configuration on bitmap creation, not on uploading to GPU. You need to set the equivalent options via `ImageBitmapLoader.setOptions()` instead. Refer to [WebGL specification](https://www.khronos.org/registry/webgl/specs/latest/1.0/#6.10) for the detail.

Reply:
Sounds good 👍!